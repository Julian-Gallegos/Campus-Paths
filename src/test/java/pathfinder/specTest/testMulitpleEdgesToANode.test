# Find a path with Dijkstra's algorithm when
# there are multiple edges from A to B
# Want to use lowest cost node

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 11.0
AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A B 5.0
AddEdge simpleGraph A B 112.0

FindPath simpleGraph A B