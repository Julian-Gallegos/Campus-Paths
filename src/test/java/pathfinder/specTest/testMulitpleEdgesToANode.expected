# Find a path with Dijkstra's algorithm when
# there are multiple edges from A to B
# Want to use lowest cost node

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 11.000 from A to B in simpleGraph
added edge 10.000 from A to B in simpleGraph
added edge 5.000 from A to B in simpleGraph
added edge 112.000 from A to B in simpleGraph

path from A to B:
A to B with weight 5.000
total cost: 5.000